src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMap.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace EA.Weee.Web.Areas.Scheme.Mappings.ToViewModels
+{
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Web.Areas.Aatf.Mappings.ToViewModel;
+    using EA.Weee.Web.Areas.Aatf.ViewModels;
+    using EA.Weee.Web.Areas.Scheme.ViewModels.ManageEvidenceNotes;
+    using System.Linq;
+
+    public class ViewAndTransferEvidenceViewModelMap : IMap<ViewAndTransferEvidenceViewModelMapTransfer, ViewAndTransferEvidenceViewModel>
+    {
```

Comment:
ViewAndTransferEvidenceViewModelMap and ReviewSubmittedEvidenceNotesViewModelMap are pretty much exactly the same. We should pull the common code (the stuff that creates the evidencenote list) into its own mapper / or service

Reply:
@NateHogan I'm going to make a change for this in the branch of code I'm working on. As can see the same code in the AATF.